src/test/epoll/test_epoll_writeable.c

```diff
@@ -2,7 +2,7 @@
  * The Shadow Simulator
  * See LICENSE for licensing information
  */
-#define _POSIX_C_SOURCE 199309L
+//#define _POSIX_C_SOURCE 199309L
```

Comment:
I know this is from my commit, but - maybe just delete? Any idea why this was here in the first place or why we might need it again?

IIRC I had to comment it out to enable some feature, but I wasn't sure commenting it out was the right fix.

Reply:
> Any idea why this was here in the first place or why we might need it again?

According to [the libc manual on Feature Test Macros](https://www.gnu.org/software/libc/manual/html_node/Feature-Test-Macros.html):

> Macro: _POSIX_C_SOURCE
>
>    Define this macro to a positive integer to control which POSIX functionality is made available. The greater the value of this macro, the more functionality is made available.
>
> ...
>
>    If you define this macro to a value greater than or equal to 199309L, then the functionality from the 1993 edition of the POSIX.1b standard (IEEE Standard 1003.1b-1993) is made available. 
> ...

There are codes for 1990, 1992, 1993, 1995, 2001, 2008. Since it works without defining it, I think deleting the line should be fine.